app/Commands/TailCommand.php

```diff
@@ -28,7 +28,7 @@ public function handle()
         $logger->addWriter($writer);
 
         $bot = new Bot();
-        $bot->setToken(config('slail.token'));
+        $bot->setToken(getenv('SLAIL_TOKEN'));
```

Comment:
I would still do this by using config('slail.token') rather than getenv()

you can set this in your service provider, I think, by replacing the putenv with config(['slail.token' => $slailConfig->token);